cypress/integration/visitorCanSeeCollectionOfArticles.feature.js

```diff
@@ -0,0 +1,20 @@
+describe('Visitor can see the main page with articles', () => {
+  describe('success', () => {
+    beforeEach(() => {
+      cy.intercept('GET', 'https://fake-newzzzz.herokuapp.com/api/articles', {
+        fixture: 'articles.json',
+      });
+      cy.visit('/');
+    });
+
+    it('loads fake articles', () => {
```

Comment:
This is not a relevant test title. We don't care if the content comes from a fixture or not.

Reply:
this is gone now